core/platform/email/mailgun_email_services_test.py

```diff
@@ -137,7 +140,8 @@ def test_reply_to_id_flag(self):
         # Lambda function, will replace post_to_mailgun().
         req_post_lambda = (lambda data=None:
```

Comment:
Perhaps break after '(' here -- the large indentation to the right is becoming awkward.

Reply:
Done